raft/raft.go

```diff
@@ -36,6 +37,17 @@ const (
 	StateLeader
 )
 
+// Possible values for CampaignType
+const (
+	LeaderElection CampaignType = "LeaderElection"
```

Comment:
I think `CampaignElection` and `CampaignTransfer` would be better names. Do these need to be exported?